entity-service/src/main/kotlin/com/egm/stellio/entity/model/EntityTypeInfo.kt

```diff
@@ -0,0 +1,18 @@
+package com.egm.stellio.entity.model
+
+import java.net.URI
+
+data class EntityTypeInfo(
+    val id: URI,
+    val type: String = "EntityTypeInformation",
+    val typeName: String,
+    val entityCount: Int,
+    val attributeDetails: List<AttributeInfo>
+)
+
+data class AttributeInfo(
```

Comment:
why not `Attribute` as is called in 5.2.28?

Reply:
There is already an `Attribute` class.

Reply:
so, I let you guess what you've forgotten :)

Reply:
I didn't get it

Reply:
Move it to the shared module ?

Reply:
when it's not "natural" (or hacky), say it in the code!